### ISSUE TITLE
[Bug]: Pod install Error

### ISSUE THREAD

**Issue by bhanuka96** [oddbit/flutter_facebook_app_events#165]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

I got this error when installing pod. I try delete my pod files but same error is coming.

`The 'Pods-Runner' target has transitive dependencies that include statically linked binaries: (.../ios/Pods/FBAudienceNetwork/Static/FBAudienceNetwork.xcframework)`

Temp Solution: downgrade plugin to `0.12.0`

### Expected Behavior

install pod without any issues

### Flutter Doctor

```shell
doctor
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 2.5.3, on macOS 12.0.1 21A559 darwin-x64, locale
    en-LK)
[✓] Android toolchain - develop for Android devices (Android SDK version 30.0.3)
[✓] Xcode - develop for iOS and macOS
[✓] Chrome - develop for the web
[✓] Android Studio (version 2020.3)
[✓] VS Code (version 1.62.1)
[✓] Connected device (2 available)
```

**Comment by zhanglijie5997** [comment]
i have  the same problem.  Is there a solution? thanks.

**Comment by bhanuka96** [comment]
@zhanglijie5997 Temp Solution: downgrade plugin to 0.12.0

**Comment by viktor-savchik-idf** [comment]
Or add following lines to your project `Podfile`

```Ruby
$static_framework = ['facebook_app_events']

pre_install do |installer|
  installer.pod_targets.each do |pod|
    if $static_framework.include?(pod.name)
      def pod.build_type;
        Pod::BuildType.static_framework
      end
    end
  end
end
```

**Comment by OneXeor** [comment]
To avoid this issue you can add following lines to your project `Podfile`

```Ruby
$static_framework = ['facebook_app_events']

pre_install do |installer|
  installer.pod_targets.each do |pod|
    if $static_framework.include?(pod.name)
      def pod.build_type;
        Pod::BuildType.static_framework
      end
    end
  end
end
```

**Comment by zooRg** [comment]
> To avoid this issue you can add following lines to your project `Podfile`
> 
> ```ruby
> $static_framework = ['facebook_app_events']
> 
> pre_install do |installer|
>   installer.pod_targets.each do |pod|
>     if $static_framework.include?(pod.name)
>       def pod.build_type;
>         Pod::BuildType.static_framework
>       end
>     end
>   end
> end
> ```

tnx this work for me

**Comment by DennisAlund** [comment]
Released in [`0.14.5`](https://pub.dev/packages/facebook_app_events/changelog#0145) 

Sorry for the delay and thank you for all the engagement!
